descriptors/ModuleDescriptor-template.json

```diff
@@ -495,7 +496,8 @@
       "subPermissions": [
         "consortia.sharing-instances.item.post",
         "consortia.sharing-instances.item.get",
-        "consortia.sharing-instances.collection.get"
+        "consortia.sharing-instances.collection.get",
+        "inventory-storage.authorities.collection.get"
```

Comment:
@RomanChernetskyi can you please update folio-spring-base version to 7.2.2 in scope of this change

Reply:
done